validator/validators/basic/basic_validator.go

```diff
@@ -80,12 +82,63 @@ func (s *BasicValidator) CheckData(startTime, endTime time.Time) error {
 				Value: aws.String(dimension.Value),
 			})
 		}
-		err := s.ValidateMetric(metric.MetricName, metricNamespace, metricDimensions, metric.MetricValue, metric.MetricSampleCount, startTime, endTime)
+		environmentValue := "Generic"
+		operationValue := "operation"
+		serviceValue := "service-name"
+
+		appSignalDimensions := []cwtypes.Dimension{
+			{
+				Name:  aws.String("HostedIn.Environment"),
+				Value: aws.String(environmentValue),
+			},
+			{
+				Name:  aws.String("Operation"),
+				Value: aws.String(operationValue),
+			},
+			{
+				Name:  aws.String("Service"),
+				Value: aws.String(serviceValue),
+			},
+		}
+
+		//App Signals metric testing (This is because we want to use a different checking method (same that was done for linux test))
```

Comment:
nit: might want to wrap it around a function instead of a comment for clean code practices. However, this is also fine